joinwiz_core/src/test/scala/joinwiz/ComputationEngineTest.scala

```diff
@@ -62,7 +62,7 @@ abstract class ComputationEngineTest[F[_]: ComputationEngine] extends AnyFunSuit
 
     left
       .leftJoin(right) {
-        case (_ wiz maybeLeft, right) => maybeLeft(_.uuid) =:= right(_.uuid)
+        case (_ wiz maybeLeft, right) => maybeLeft((e: Entity) => e.uuid) =:= right(_.uuid)
```

Comment:
why adding a type hint there?

Reply:
Scala 2.11 compiler cannot infer that type and compilation fails